variables.tf

```diff
@@ -140,3 +140,9 @@ variable "nlb_logs_prefixes" {
   default     = ["nlb"]
   type        = list(string)
 }
+
+variable tags {
+  type        = map(string)
+  default     = {}
+  description = "A mapping of tags to assign to the logs bucket."
```

Comment:
You may have mentioned this in slack already...why did you choose a map of strings again? Is that the typical structure for tags or something?

Reply:
Tags, as setup within an `aws_s3_bucket` resource, already looks like a map of strings where you have a collection of key-value pairs. It made sense to me to have the variable `tags` (the optional tags a user can specify in a module) also be a map of strings. This also allows these optional tags to be merged into the default tags without doing any transformations.